### PR TITLE
Angus/catalog info strings

### PR DESCRIPTION
**Pull request by veggiesaurus**
Adds additional catalog info in the file info description field:
- Column count
- Row count (if it is defined)
- Parameters (from VOTable `PARAM` nodes)
- Coordinate system info (from VOTable `COOSYS` node)

### CODE REVIEW

**Review by markccchiang — APPROVED** [review]

File: Table/Table.cc [hunk 1]
```diff
@@ -216,28 +281,29 @@ bool Table::ConstructFromFITS(bool header_only) {
...
-#pragma omp parallel for default(none) schedule(dynamic) shared(num_cols, buffer, _num_rows, total_width)
+#pragma omp parallel for default(none) schedule(dynamic) shared(num_cols, buffer, allocated_rows, total_width)
         for (auto i = 0; i < num_cols; i++) {
-            _columns[i]->FillFromBuffer(buffer.get(), _num_rows, total_width);
+            _columns[i]->FillFromBuffer(buffer.get(), allocated_rows, total_width);
         }
     } else {
         fits_close_file(file_ptr, &status);
     }
+    _num_rows = rows;
```
Comment (confluence) [inline Table/Table.cc]: Is this intentionally set to `rows` and not `allocated_rows`?

Comment (veggiesaurus) [reply Table/Table.cc]: yes. `allocated_rows` is the number of rows to allocate for reading. When reading the header, this is zero. I'll add a comment

Comment (veggiesaurus) [reply Table/Table.cc]: actually, this is misleading. We need a new field to indicate the total rows available, not the total rows read into the table

**Comment by veggiesaurus** [comment]
@confluence please have another look, I've made some adjustments

**Review by confluence — APPROVED** [review]

**Pull request merged**